x-pack/plugins/security_solution/public/resolver/view/resolver_without_providers.tsx

```diff
@@ -16,13 +16,14 @@ import { EdgeLine } from './edge_line';
 import { GraphControls } from './graph_controls';
 import { ProcessEventDot } from './process_event_dot';
 import { useCamera } from './use_camera';
-import { SymbolDefinitions, useResolverTheme } from './assets';
+import { SymbolDefinitions } from './symbol_definitions';
```

Comment:
:grey_question: Is there a way to move this to an even higher level out of the Resolver (so we'd have one shared `<defs>` for all Resolvers?) As it stands, the way I read this we'd still have 1 defs per Resolver.

Reply:
yes. we can have a 'provider` that renders the SVG elements and defines the IDs. It would then provide those via a react `context`. 

If Resolver was rendered without the provider, it would create its own SVGs. it if was rendered as a descendant of a provider it would use the provided ones.

I didn't implement this in this PR though.

Reply:
Sounds good, probably doesn't have any kind of real impact for now anyway just wanted to keep a chip on that square for the future.